Firestore/Protos/CMakeLists.txt

```diff
@@ -306,3 +326,95 @@ add_custom_target(
+      ${protobuf_objc_source_dir}/GPBRuntimeTypes.h
+      ${protobuf_objc_source_dir}/GPBUnknownField.h
+      ${protobuf_objc_source_dir}/GPBUnknownFieldSet.h
+      ${protobuf_objc_source_dir}/GPBUnknownFieldSet_PackagePrivate.h
+      ${protobuf_objc_source_dir}/GPBUnknownField_PackagePrivate.h
+      ${protobuf_objc_source_dir}/GPBUtilities.h
+      ${protobuf_objc_source_dir}/GPBUtilities_PackagePrivate.h
+      ${protobuf_objc_source_dir}/GPBWellKnownTypes.h
+      ${protobuf_objc_source_dir}/GPBWireFormat.h
+      ${protobuf_objc_source_dir}/google/protobuf/Any.pbobjc.h
+      ${protobuf_objc_source_dir}/google/protobuf/Api.pbobjc.h
+      ${protobuf_objc_source_dir}/google/protobuf/Duration.pbobjc.h
+      ${protobuf_objc_source_dir}/google/protobuf/Empty.pbobjc.h
+      ${protobuf_objc_source_dir}/google/protobuf/FieldMask.pbobjc.h
+      ${protobuf_objc_source_dir}/google/protobuf/SourceContext.pbobjc.h
+      ${protobuf_objc_source_dir}/google/protobuf/Struct.pbobjc.h
+      ${protobuf_objc_source_dir}/google/protobuf/Timestamp.pbobjc.h
+      ${protobuf_objc_source_dir}/google/protobuf/Type.pbobjc.h
+      ${protobuf_objc_source_dir}/google/protobuf/Wrappers.pbobjc.h
+
+    SOURCES
+      ${protobuf_objc_source_dir}/GPBArray.m
+      ${protobuf_objc_source_dir}/GPBCodedInputStream.m
+      ${protobuf_objc_source_dir}/GPBCodedOutputStream.m
+      ${protobuf_objc_source_dir}/GPBDescriptor.m
+      ${protobuf_objc_source_dir}/GPBDictionary.m
+      ${protobuf_objc_source_dir}/GPBExtensionInternals.m
+      ${protobuf_objc_source_dir}/GPBExtensionRegistry.m
+      ${protobuf_objc_source_dir}/GPBMessage.m
+      ${protobuf_objc_source_dir}/GPBProtocolBuffers.m
+      ${protobuf_objc_source_dir}/GPBRootObject.m
+      ${protobuf_objc_source_dir}/GPBUnknownField.m
+      ${protobuf_objc_source_dir}/GPBUnknownFieldSet.m
+      ${protobuf_objc_source_dir}/GPBUtilities.m
+      ${protobuf_objc_source_dir}/GPBWellKnownTypes.m
+      ${protobuf_objc_source_dir}/GPBWireFormat.m
+      ${protobuf_objc_source_dir}/google/protobuf/Any.pbobjc.m
+      ${protobuf_objc_source_dir}/google/protobuf/Api.pbobjc.m
+      ${protobuf_objc_source_dir}/google/protobuf/Duration.pbobjc.m
+      ${protobuf_objc_source_dir}/google/protobuf/Empty.pbobjc.m
+      ${protobuf_objc_source_dir}/google/protobuf/FieldMask.pbobjc.m
+      ${protobuf_objc_source_dir}/google/protobuf/SourceContext.pbobjc.m
+      ${protobuf_objc_source_dir}/google/protobuf/Struct.pbobjc.m
+      ${protobuf_objc_source_dir}/google/protobuf/Timestamp.pbobjc.m
+      ${protobuf_objc_source_dir}/google/protobuf/Type.pbobjc.m
+      ${protobuf_objc_source_dir}/google/protobuf/Wrappers.pbobjc.m
+  )
+  target_include_directories(
+    Protobuf
+    PRIVATE
+      ${protobuf_objc_source_dir}
+      ${protobuf_objc_source_dir}/google/protobuf
+    INTERFACE
+      ${PROJECT_BINARY_DIR}/Headers/Protobuf
+      ${protobuf_objc_source_dir}
```

Comment:
nit: PUBLIC effectively means PRIVATE+INTERFACE, so I think this could become:
```
  target_include_directories(
￼     Protobuf
￼     PRIVATE
￼       ${protobuf_objc_source_dir}/google/protobuf
￼     INTERFACE
￼       ${PROJECT_BINARY_DIR}/Headers/Protobuf
     PUBLIC
￼       ${protobuf_objc_source_dir}
  )
```
Optional.

Reply:
Done.